edx-platform/nau-basic/conf/locale/babel_mako.cfg

```diff
@@ -6,6 +6,14 @@ input_encoding = utf-8
 # [django: **/templates/**.html]
 # input_encoding = utf-8
 
+# Use to target ace_common django template files
```

Comment:
Is this target necessary?

Reply:
Yes, because the message we want to translate is in `templates/ace_common/edx_ace/common/nudge_email_unsubscribe_message.html`, but I remove the schedule target. Thanks ✨

Reply:
I understand, thanks!